### PR TITLE
Fix pointer direction when collapsed by add a css transform

### PR DESCRIPTION
**Pull request by ari7946**

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-1)

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-1)

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-1)

File: features/ui/sidebar-navigation/sidebar-navigation.tsx [hunk 1]
```diff
@@ -144,8 +144,9 @@ const LinkList = styled(List)`
   flex: 1;
 `;
 
-const CollapseMenuItem = styled(MenuItemButton)`
+const CollapseMenuItem = styled(MenuItemButton)<{ isCollapsed: boolean }>`
   display: none;
+  transform: ${({ isCollapsed }) => (isCollapsed ? "rotate(180deg)" : "none")};
```
Comment (jkettmann) [inline features/ui/sidebar-navigation/sidebar-navigation.tsx:149]: Exactly how I would do it

**Pull request merged**